admin-dev/themes/new-theme/js/components/form/form-field-disabler.ts

```diff
@@ -27,6 +27,12 @@ import {isUndefined} from '@PSTypes/typeguard';
 
 const {$} = window;
 
+// eslint-disable-next-line no-shadow
+export enum toggleType {
+  visibility = 'visibility',
+  availability = 'availability',
```

Comment:
```suggestion
  visible = 'visible',
  disable = 'disable',
```
Just a suggestion, even though your naming is semantically more correct I think about this alternative which may be easier to understand at first read. If you prefer yours we use yours though.

Reply:
according to you then it should become "visbile|enabled" or "invisible|disabled", but I think I'll leave it my way, as you said its semantically more correct :thinking: